### PR TITLE
feat: Azure stack e2e changes

### PR DESCRIPTION
**Pull request by Bhuvaneswari-Santharam**
**Reason for Change**:

This allows us to run e2e test on AzureStack

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: test/e2e/config/config.go [hunk 1]
```diff
@@ -57,17 +81,153 @@ func ParseConfig() (*Config, error) {
...
+	}
+	return ccc, nil
+}
+
 // GetKubeConfig returns the absolute path to the kubeconfig for c.Location
 func (c *Config) GetKubeConfig() string {
 	var kubeconfigPath string
 
-	if c.IsKubernetes() {
+	switch {
```
Comment (honcao) [inline test/e2e/config/config.go]: there is only one case, so do not need the switch.

**Comment by CecileRobertMichon** [comment]
Unit tests failed on lint:

```
test/e2e/runner.go:40: File is not `goimports`-ed (goimports)
	
        if cfg.IsAzureStackCloud() {
test/e2e/config/config.go:18: File is not `goimports`-ed (goimports)
         
test/e2e/kubernetes/kubernetes_suite_test.go:10: File is not `goimports`-ed (goimports)
	"github.com/onsi/ginkgo/reporters"
Makefile:67: recipe for target 'validate-go' failed
```

**Comment by jadarsie** [comment]
@Bhuvaneswari-Santharam, could you please make the PR title more specific? it does not provide much info as it is.

Also, I think it should be preffixed with `test:` instead of `feat:`

**Pull request closed without merging**